packages/create-workleap/package.json

```diff
@@ -26,10 +26,5 @@
         "@clack/core": "0.3.2",
         "@clack/prompts": "0.6.3",
         "picocolors": "1.0.0"
```

Comment:
I'm not so sure about that. I was looking at Chakra UI, and they have tsup at the root only, instead of in each packages.

I would prefer having it at the root only, since i want this dependency to be the same across all my packages